packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -115,64 +127,162 @@ class Accordion extends AutoControlledComponent<WithAsProp<AccordionProps>, any>
   }
 
   public static defaultProps = {
-    accessibility: defaultBehavior as Accessibility,
+    accessibility: accordionBehavior,
+    as: 'dl',
   }
 
   static autoControlledProps = ['activeIndex']
 
   static Title = AccordionTitle
   static Content = AccordionContent
 
-  getInitialAutoControlledState({ exclusive }) {
-    return { activeIndex: exclusive ? -1 : [-1] }
+  private focusHandler: ContainerFocusHandler = null
+  private itemRefs = []
+
+  actionHandlers: AccessibilityActionHandlers = {
+    moveNext: e => {
+      e.preventDefault()
+      this.focusHandler.moveNext()
+    },
+    movePrevious: e => {
+      e.preventDefault()
+      this.focusHandler.movePrevious()
+    },
+    moveFirst: e => {
+      e.preventDefault()
+      this.focusHandler.moveFirst()
+    },
+    moveLast: e => {
+      e.preventDefault()
+      this.focusHandler.moveLast()
+    },
+  }
+
+  constructor(props, context) {
+    super(props, context)
+
+    this.focusHandler = new ContainerFocusHandler(
+      this.getNavigationItemsSize,
+      this.handleNavigationFocus,
+      true,
+    )
+  }
+
+  private handleNavigationFocus = (index: number) => {
+    this.setState({ focusedIndex: index }, () => {
+      const targetComponent = this.itemRefs[index] && this.itemRefs[index].current
+      const targetDomNode = ReactDOM.findDOMNode(targetComponent) as any
+
+      targetDomNode && targetDomNode.focus()
```

Comment:
No need for using ReactDOM here. You can directly use the targerComponent.focus(). Is there a case where this doesn't work? I've tested this and it seems to work, but maybe I am missing something...

```
 private handleNavigationFocus = (index: number) => {
    this.setState({ focusedIndex: index }, () => {
      const targetComponent = this.itemRefs[index] && this.itemRefs[index].current

      targetComponent && targetComponent.focus()
    })
  }
```

Reply:
copy-pasted this from List. Will change.